### PR TITLE
350 Product category card from crosswalk index

### LINKED ISSUE #350: [Product Category] INDEX Category List (Crosswalk Index)

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #350 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://350-category-list-xwalk-index--danaher-ls-aem--hlxsites.hlx.page/us/en/products/centrifuges
- After: https://350-category-list-xwalk-index--danaher-ls-aem--hlxsites.hlx.page/us/en/products/microscopes

### CODE REVIEW

File: blocks/product-category/product-category.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  );
+  return li({
+    class:
+        'w-full flex flex-col col-span-1 relative mx-auto justify-center transform transition duration-500 border hover:scale-105 shadow-lg rounded-lg overflow-hidden bg-white max-w-xl',
+  }, cardWrapper);
+}
+
+export default async function decorate(block) {
+  const categories = window.location.pathname.slice(16).split('.').at(0).split('/');
+  const categoty = categories[1] ? categories[1] : categories[0];
```
Comment (mhaack) [inline blocks/product-category/product-category.js:31]: Using 16 as fixed numbers seams easy to break? Couldn't we just split by `/` and work with .pop()?

File: blocks/product-category/product-category.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  const products = await ffetch('/us/en/products-index.json')
+    .filter(({ parentCategory }) => parentCategory.toLowerCase() === categoty.toLowerCase())
```
Comment (mhaack) [inline blocks/product-category/product-category.js:34]: Do we need to apply some alphabetical sorting?

**Review by davenichols-DHLS — APPROVED** [review]
Love it, thanks Ravi.  If possible can we add the title and text from the banner on the child page to get this data in the index (FYI @mhaack ).

File: blocks/product-category/product-category.js (hunk 1 above)
Comment (duynguyen) [reply blocks/product-category/product-category.js:31]: there's also a `category` meta in the page head. can it be used here?

Comment (rgravitvl) [reply blocks/product-category/product-category.js]: Done

File: blocks/product-category/product-category.js (hunk 2 above)
Comment (rgravitvl) [reply blocks/product-category/product-category.js]: Done

**Review by mhaack — APPROVED** [review]
LGTM now. The test pages are missing title & description but I think that is a content issue.

**Pull request merged**